src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -292,35 +310,19 @@ class SaveInProgressIntro extends React.Component {
     }
 
     if (this.props.startMessageOnly && !savedForm) {
-      return <div>{this.getAlert(savedForm)}</div>;
+      return <div>{this.getAlert(savedForm).alert}</div>;
     }
 
+    const { alert, includesFormControls } = this.getAlert(savedForm);
+
     const content = (
       <div>
-        {!this.props.buttonOnly && this.getAlert(savedForm)}
-        {this.props.buttonOnly &&
-          !this.props.user.login.currentlyLoggedIn &&
-          this.getAlert(savedForm)}
-        {this.props.user.login.currentlyLoggedIn && (
-          <FormStartControls
-            resumeOnly={this.props.resumeOnly}
-            isExpired={isExpired}
-            messages={this.props.messages}
-            startText={this.props.startText}
-            testActionLink={this.props.testActionLink}
-            startPage={startPage}
-            formId={this.props.formId}
-            returnUrl={this.props.returnUrl}
-            migrations={this.props.migrations}
-            prefillTransformer={this.props.prefillTransformer}
-            fetchInProgressForm={this.props.fetchInProgressForm}
-            removeInProgressForm={this.props.removeInProgressForm}
-            prefillAvailable={prefillAvailable}
-            formSaved={!!savedForm}
-            gaStartEventName={this.props.gaStartEventName}
-          />
-        )}
-        {!this.props.buttonOnly && this.props.afterButtonContent}
+        {!buttonOnly && alert}
+        {buttonOnly && !login.currentlyLoggedIn && alert}
+        {!includesFormControls &&
+          login.currentlyLoggedIn &&
+          this.getFormControls(savedForm)}
```

Comment:
This is related to my earlier comment about the form control being rendered twice. I see that the `!includesFormControls1` conditional would not let that happen :+1: